packer/db.pkr.hcl

```diff
@@ -0,0 +1,57 @@
+variable "instance_size" { type = string }
+variable "region" { type = string }
+variable "base_ami" { type = string }
+
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.2.8"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "db" {
+  ami_name      = "ami-db"
+  instance_type = var.instance_size
+  region        = var.region
+  source_ami    = var.base_ami
+  ssh_username  = "ubuntu"
+  tags = {
+    Project = "Homework-4"
+  }
+}
+
+build {
+  name = "db"
+  sources = [
+    "source.amazon-ebs.db"
+  ]
+  provisioner "file" {
+    source      = "./services/backup.service"
+    destination = "/tmp/backup.service"
+  }
+  provisioner "file" {
+    source      = "./services/backup.timer"
+    destination = "/tmp/backup.timer"
+  }
+  provisioner "file" {
+    source      = "./scripts/backup.sh"
+    destination = "/home/ubuntu/backup.sh"
+  }
+  provisioner "shell" {
+    inline = [
+      "sudo apt update",
+      "sudo apt install -y mariadb-server awscli"
+    ]
+    pause_before = "10s"
+  }
+  provisioner "shell" {
```

Comment:
If you need to pass packer variables to script you can use `environment_vars` section:
```
  provisioner "shell" {
    environment_vars = [
      "REGION=${var.region}"
    ]
    ...
```

but if you want to run script with `sudo` it'll be a bit tricky:
```
  provisioner "shell" {
    environment_vars = [
      "REGION=${var.region}"
    ]
    script = "./scripts/example.sh"
    execute_command = "sudo bash -c '{{.Vars}} {{.Path}}'"
  }
```
  
execute_command = "sudo bash -c '{{.Vars}} {{.Path}}'"

Reply:
Understood, I'll remember that.Thanks for the comment!